packages/components/src/internal/components/buttons/ManageDropdownButton.test.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import { ManageDropdownButton } from './ManageDropdownButton';
+
+describe('ManageDropdownButton', () => {
+    test('default props', () => {
+        const { container } = render(<ManageDropdownButton />);
+        expect(container).toMatchSnapshot();
```

Comment:
I switched them away from snapshot tests. I do think that these very simple components are a valid case for snapshot tests though since it is very clear when looking at the simple DOM when something changes to determine if that is an expected change or not.